src/components/List.component.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// TODO: discuss if there are no outer dependecies should we wrap component into a function
+const List = (props) => {
+    return (
+        <div>
+            <ul>
+                {
+                    props.listItems.map((el) => (
+                        <li
+                            key={el.id}>
+                            <div className="form-inline">
+                                <div className="form-group">
+                                    <a href="javascript: void(0)" onClick={() => props.changeRoute(`/${el.cityCode}`)}>
```

Comment:
Arrow functions in jsx are considered a bad practice: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md#protips. Should we extract all of them to a separate private function though there will be no other code there?